admin/services/check-window.service.js

```diff
@@ -2,13 +2,31 @@
 const CheckWindow = require('../models/check-window')
 
 const checkWindowService = {
-  getCurrentCheckWindow: async function () {
+
+  /**
+   * Get check window for the current time.
+   * @returns {Promise.<*>}
+   */
+  getCurrentCheckWindow: async () => {
     const now = new Date()
     const checkWindow = await CheckWindow.findOne({startDate: {$lte: now}, endDate: {$gte: now}}).exec()
     if (!checkWindow) {
       throw new Error('No checkwindow is currently available')
```

Comment:
I don't see a need to throw an error here.  Isn't returning null sufficient?  I don't think we can predict what the caller expects here.  Conversely, by throwing here the caller must always use a try / catch block - that seems sub-optimal to me.

Reply:
OK, I'll remove it